app/helpers/application_helper.rb

```diff
@@ -1,2 +1,7 @@
 module ApplicationHelper
+
+  def select_type(skill_type)
+    @character.skills.order(:name).select{ |type_skill| type_skill.type_of_skill == skill_type }
```

Comment:
Btw two more things

- Don't use instance variables on helper methods, it can be a bit hard to debug (since you have to explicitly know where the instance variable is coming from) and, if another view want to use this method it'll have to define the same instance variable
- Change the name to something more descriptive `select_type` is too general